axis.go

```diff
@@ -407,14 +418,63 @@ func (DefaultTicks) Ticks(min, max float64) (ticks []Tick) {
 		if val >= min && val <= max && !found {
 			ticks = append(ticks, Tick{Value: val})
 		}
-		if math.Nextafter(val, val+minorDelta) == val {
-			break
-		}
 		val += minorDelta
 	}
 	return
 }
 
+func removeDuplicates(elements []float64) []float64 {
+	// Uses a map to record duplicates as we find them.
+	encountered := map[float64]bool{}
+	result := []float64{}
+
+	for v := range elements {
```

Comment:
I'd use the key,value range instead:

```go
for i,v := range elements {
    if encountered[v] {
        continue
    }
    encountered[v] = true
    results = append(results, v)
}
return result
```

also, more Go-idiomatic names would be:
```go
seen := make(map[float64]bool)
out := make([]float64, 0, len(elements))
```

Reply:
Changed that. Thanks!